modules/serverless-init-containers-apps.adoc

```diff
@@ -1,47 +0,0 @@
-// Module included in the following assemblies:
```

Comment:
Is the removal of multiple blocks an intended change?

Reply:
@ReToCode It is true that some are not related to the topic of this PR, I should probably remove them separately. If you are curious, the selection of modules I deleted is the one returned by the script for finding unused modules. (`repo/scripts/find_unused.py`)